core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingRepository.scala

```diff
@@ -98,23 +98,14 @@ class BillingRepository(dataSource: SlickDataSource) {
   def failUnlessHasNoWorkspaces(projectName: RawlsBillingProjectName)(implicit ec: ExecutionContext): Future[Unit] =
     dataSource.inTransaction { dataAccess =>
       dataAccess.workspaceQuery.listWithBillingProject(projectName) map { workspaces =>
-        val (v1Workspaces, v2Workspaces) = workspaces.partition(ws => ws.workspaceVersion == WorkspaceVersions.V1)
-        v2Workspaces map { _ =>
+        workspaces map { _ =>
           throw new RawlsExceptionWithErrorReport(
             ErrorReport(
               StatusCodes.BadRequest,
               "Project cannot be deleted because it contains workspaces."
             )
           )
         }
-        v1Workspaces map { _ =>
-          throw new RawlsExceptionWithErrorReport(
-            ErrorReport(
-              StatusCodes.BadRequest,
-              "Project cannot be deleted because it contains v1 workspaces. Contact support for help."
```

Comment:
It's likely that Sam will prevent this as well as the `billing-project` `owner` policy group is a member of the `project-owner` `workspace` policy group.

Reply:
Bah: you're right...
When I pulled out this part, I was thinking it would still get the workspaces, and still error.
But I also changed the query to only return v2 workspaces.
This is the one place where we still need to get v1 workspaces.

Reply:
I removed the version from the query, so this should still throw on v1 workspaces, now (I never changed the places that were filtering them out manually anyway).